### PR TITLE
Create a 404 page.

### PR DESCRIPTION
**Pull request by neelxie**
#### What does this PR do?
It creates a 404 page for the application incase an unknown URL is entered.

#### Screenshots

Screenshot from 2020-12-22 11-12-24

### CODE REVIEW

**Review by MutegekiHenry — DISMISSED** [review]

File: src/components/404page/404page.css [hunk 1]
```diff
@@ -0,0 +1,40 @@
+.ErrorPageContainer{
```
Comment (StephenAraka) [inline src/components/404page/404page.css:1]: This is good... however, `flexbox` with `justify-content` and `align-items` would've been better... especially considering the conventions we have been discussing.

**Review by StephenAraka — CHANGES_REQUESTED** [review]
Good job on this one. 
Just a few notes:
- Checkout the suggested changes on the comment made on the `404page.css`.
- Remove the full-stop on the PAGE NOT FOUND. It's not there on the design.

**Comment by neelxie** [comment]
> Good job on this one.
> Just a few notes:
> 
> 1. Checkout the suggested changes on the comment made on the `404page.css` - could work on them now or later (depends) coz the page looks just fine.
> 2. Remove the full-stop on the PAGE NOT FOUND. It's not there on the design.

@StephenAraka this feedback has been worked upon, please do check it out.

**Review by StephenAraka — APPROVED** [review]

**Comment by StephenAraka** [comment]
Well done.

**Review by MutegekiHenry — APPROVED** [review]

**Pull request merged**